examples/securenn-playground.py

```diff
@@ -7,14 +7,14 @@
     'crypto_producer'
 ])
 
-with tfe.protocol.SecureNN(*config.get_players('server0, server1, crypto_producer')) as prot:
+with tfe.protocol.SecureNN(*config.get_players('server0, server1, crypto_producer'), use_native=True) as prot:
 
     a = prot.define_constant(np.array([0, 0, 1, 1]), apply_scaling=False)
     b = prot.define_constant(np.array([0, 1, 0, 1]), apply_scaling=False)
     c = prot.bitwise_or(a, b)
 
-    x = prot.define_constant(np.array([0., 1., 2., 3.]))
-    y = prot.define_constant(np.array([0., 1., 2., 3.]))
+    x = prot.define_constant(np.array([0., 1., 2., 3.]), apply_scaling=False)
```

Comment:
scaling should be applied here and below, any reason for changing it?

Reply:
I was just trying to get my weird hack to return the right numbers and adding `apply_scaling=False`, same reason I added the bound hack below. Figured its not the right way of doing things.